src/python/WMComponent/JobAccountant/AccountantWorker.py

```diff
@@ -887,6 +894,8 @@ def createFileFromDataStructsFile(self, fname, jobID):
         wmbsFile["locations"] = set()
 
         if pnn != None:
+            if pnn == "T1_US_FNAL":
+                pnn = "T1_US_FNAL_Disk"
```

Comment:
Todor, can you please add a logging.warning here printing the job id that had this broken PNN?
We had new cases and Farrukh would be interested in knowing which job ids (actually which worker node) is still pointing to the wrong SITECONF.

Please provide a new commit, such that we can easily apply only the extra commit.